java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1,15 +1,14 @@
 package jmri.jmrit.display.layoutEditor;
 
-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
 import static jmri.util.MathUtil.add;
```

Comment:
I *strongly recommend* that you not use static imports for methods.

From [Java Platform Standard Edition 8 Documentation > Static Import](http://docs.oracle.com/javase/8/docs/technotes/guides/language/static-import.html):

> So when should you use static import? **Very sparingly!** Only use it when you'd otherwise be tempted to declare local copies of constants, or to abuse inheritance (the Constant Interface Antipattern). In other words, use it when you require frequent access to static members from one or two classes. If you overuse the static import feature, it can make your program unreadable and unmaintainable, polluting its namespace with all the static members you import. Readers of your code (including you, a few months after you wrote it) will not know which class a static member comes from. Importing all of the static members from a class can be particularly harmful to readability; if you need only one or two members, import them individually.

(Emphasis theirs)